src/delaunay.c

```diff
@@ -382,7 +404,9 @@ int del_layer_delaunay_reduce_2D(double red_lattice[3][3],
 
     get_extended_basis_2D(basis, lattice_2D);
 
-    for (attempt = 0; attempt < NUM_ATTEMPT; attempt++) {
+    for (attempt = 0; attempt < get_num_attempts(); attempt++) {
```

Comment:
```suggestion
    for (int attempt = 0, max_attempt = get_num_attempts();
         attempt < max_attempt; attempt++) {
```

Reply:
Same as the above.

Reply:
Oh, forgot to look for other sources, let me add some small adjustments